specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -452,19 +459,29 @@
               "$ref": "#/definitions/Vault"
             }
           },
-          "201": {
-            "description": "Created",
-            "schema": {
-              "$ref": "#/definitions/Vault"
-            }
+          "202": {
```

Comment:
What's the reason behind updating the response code?

Reply:
The reason is that 201 was not being used from the service side. The next reason is that the synchronous call has been made Asynchronous so 202 is added as per the ARM guidelines

Reply:
Good, I realized it after looking at the new operationStatus endpoints but it wasn't clear from the PR description nor from the title.

Reply:
Sorry for the confusion.